server/lib/search.js

```diff
@@ -86,11 +86,15 @@ export const searchCollectivesInDB = async (
   term,
   offset = 0,
   limit = 100,
-  { types, hostCollectiveIds, isHost, onlyActive, skipRecentAccounts, hasCustomContributionsEnabled } = {},
+  { types, hostCollectiveIds, isHost, onlyActive, skipRecentAccounts, hasCustomContributionsEnabled, countries } = {},
 ) => {
   // Build dynamic conditions based on arguments
   let dynamicConditions = '';
   let isUsingTsVector = false;
+  let countryCodes = 'ALL';
```

Comment:
Our generic practice is to use a `null` value instead of a `'ALL'` string like here.

Reply:
I've changed this to use `null`. Thanks. 👍🏽